### PR TITLE
chore: release docs and cp readmes

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
Closes out #346 with release updates. 

Includes:

- [x] Add build step for homestar-runtime npm wrapper (cc @hugomrdias).
- [X] Makes nix build of homestar part of `push` event on `main` branch. 
- [X] Local build optimization(s) with [cargo-hakari](https://crates.io/crates/cargo-hakari). 
- [X] Graceful test drop, specified for unix/windows
- [X] Local windows-cross target
- [X] Fix GH links for releases on crates.io.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,15 +167,50 @@ We have a focused [development](./DEVELOPMENT.md) guide, as well as a
...
 We always adhere to our [Code of Conduct](./CODE_OF_CONDUCT.md).
 
-## Releases
+## Releases and Builds
 
-TBA
+### Crates, Tags, and GitHub Releases
+
+Homestar uses [release-plz][release-plz] to publish [crates][rel-crates],
+[tags][rel-tags], changelogs, and [GitHub Releases][rel-gh]. Upon merging
```
Comment (bgins) [inline README.md:175]:
```suggestion
[tags][rel-tags], changelogs, and [GitHub Releases][rel-gh]. Upon merging,
```

File: SECURITY.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ should not be reported via the public GitHub Issue tracker.
 
 The project team is committed to transparency in the security issue disclosure
 process. The Fission team announces security advisories through our
-Github respository's [security portal][sec-advisories] and and the
+GitHub respository's [security portal][sec-advisories] and and the
```
Comment (bgins) [inline SECURITY.md:12]:
```suggestion
GitHub respository's [security portal][sec-advisories] and the
```

**Review by bgins — APPROVED** [review]
Looks good! Left a couple of minor copy edit suggestions.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -50,6 +50,11 @@ jobs:
       - name: Cache Project
         uses: Swatinem/rust-cache@v2
 
+      - name: Install cargo-hakari
```
Comment (zeeshanlakhani) [inline .github/workflows/release.yml:53]: remove this.

**Pull request merged**